mne/time_frequency/multitaper.py

```diff
@@ -543,12 +542,70 @@ def multitaper_psd(x, sfreq=2 * np.pi, fmin=0, fmax=np.inf, bandwidth=None,
                        for x in np.array_split(x_mt, n_jobs))
         psd = np.concatenate(out)
 
-    if x.ndim == 1:
-        # return a 1d array if input was 1d
-        psd = psd[0, :]
-
     freqs = freqs[freq_mask]
     if normalization == 'full':
         psd /= sfreq
 
+    # Combining/reshaping to original data shape
+    psd = psd.reshape(np.hstack([dshape, -1]))
+    if ndim_in == 1:
+        psd = psd.squeeze()
     return psd, freqs
+
+
+@deprecated('This will be deprecated in release v0.13, see _psd_multitaper.')
```

Comment:
I was wondering about that. On the one hand `psd_multitaper` is the new public function, on the other hand it doesn't duplicate the functionality of `multitaper_psd` since it won't accept arrays. You think that's fine?